src/schemas/modelMonitor.py

```diff
@@ -1,14 +1,37 @@
-from typing import Any, Optional, Union
 from pydantic import BaseModel
+import enum
 from src.schemas.base import ItemBase
 
 
+class MonitorStatus(str, enum.Enum):
+    open = "open"
+    closed = "closed"
+
+
+class MonitorMetrics(str, enum.Enum):
+    accuracy = "accuracy"
+    precision = "precision"
+    recall = "recall"
+    f1 = "f1"
+    data_drift = "data_drift"
+    concept_drift = "concept_drift"
+    missing_values_count = "missing_values_count"
+
+
+class AlertSeverity(str, enum.Enum):
+    low = "low"
+    mid = "mid"
+    high = "high"
+
+
 class ModelMonitorBase(BaseModel):
     model_id: str
-    dataset_id: str
     name: str
-    # metric: Any #Enum
-    threshold: Union[int, float]
+    status: MonitorStatus
+    metric: MonitorMetrics
+    threshold: float
+    severity: AlertSeverity
+    email: str
```

Comment:
Some food for though on this is that the drifting in multiclass models is measured per class.
As is missing values count in general.